src/jukebox/components/playermpd/coverart_cache_manager.py

```diff
@@ -1,9 +1,13 @@
 import os
+import jukebox.cfghandler
+
+cfg = jukebox.cfghandler.get_handler('jukebox')
 
 
 class CoverartCacheManager:
-    def __init__(self, cache_folder_path):
-        self.cache_folder_path = cache_folder_path
+    def __init__(self):
+        coverart_cache_path = cfg.setndefault('webapp', 'coverart_cache_path', value='../../src/webapp/build/cover-cache')
```

Comment:
I'm wondering is there a use case where a user needs that path configurable?

Reply:
The question applies on many vars in the yaml.

I think we can think of the yaml as a global var handler.

Reply:
> I'm wondering is there a use case where a user needs that path configurable?

It's not required to be configurable, especially now that it is handled in its own class. Yet, it's a best practice at the moment. We might want to review the `jukebox.yaml` and check, if this could be done differently!